docs/collaboration.md

```diff
@@ -0,0 +1,105 @@
+
+The Configuration file describes how the content will be deployed to Posit
+Connect.
+
+An example of a Configuration file is `/.posit/publish/content-title-061E.toml`.
+
+For more details on the Configuration file see the [Configuration file reference](./configuration.md).
+
+#### The Deployment file
+
+The Deployment file contains details on where the content was deployed and its
+state. Deployment files are automatically generated by Posit Publisher,
+updated during deploys, and not meant to be edited.
+
+An example of a Deployment file is
+`/.posit/publish/deployments/deployment-58A3.toml`.
+
+Utilizing both of these files, alongside a credential and collaborator access
+on Posit Connect, you and your collaborators can update and configure a single
+piece of content.
+
+## Collaborator access
+
+The original creator of a piece of content on Posit Connect needs to give others
+collaborator access to update the content.
+
+See the [Posit Connect User Guide - Access list settings](https://docs.posit.co/connect/user/content-settings/#set-collaborators)
+documentation for more information on giving users access to update a piece of
+content.
+
+## Source control setup
+
+Deployment and Configuration files found in `./posit/publish` are intended
+to be committed into source repositories.
+
+Once tracked in source control, anyone with access can easily collaborate on the
+same piece of content.
+
+After a deploy be sure to commit any changes to the Deployment and Configuration
+files. Keeping them in sync with the content on Posit Connect will ensure that
+collaborators have the most up-to-date details to continue where others left
+off.
+
+## Content bundle setup
+
+If you and your collaborators aren't using source control, a content bundle
+containing the source code and data for a piece of content can be downloaded
+from Posit Connect. Content bundles are created when content is published and
+they work great with Posit Publisher.
+
+When deploying, include the Deployment and Configuration files so others can
+use them to collaborate after they download the content bundle.
+
+Posit Publisher includes both files automatically in configurations it generates
+to make this effortless.
+
+See the
+[Posit Connect User Guide - Source Versions](https://docs.posit.co/connect/user/source-versions/)
+documentation for more information on the content bundle source versions and
+how to download them.
+
+> [!NOTE]
+> The content bundle will only contain files sent to Posit Connect. Be sure any
+> files that you would like collaborators to have access to are in the
+> Configuration's included `files`.
+> For more details see the [Project Files documentation for Posit Publisher](./vscode.md#project-files).
+
+## Tips
+
+### Stay updated while collaborating
+
+While collaborating on a piece of content, stay updated with the latest changes
+others have made.
+
+To do so, either pull down the latest changes from the repository or download
```

Comment:
```suggestion
To do so, either pull down the latest changes from your source control system or download
```

Most (all?) other places we call this source control and I think it would be good to keep it here for consistency (and it avoids ambiguity around the word `repository`)

Reply:
Replaced in https://github.com/posit-dev/publisher/pull/2507/commits/de88b380ecec4692ae69ab7a9e71b080eb6927b3. 👍 great suggestion.